frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx

```diff
@@ -57,9 +62,17 @@ const InstallCluster: React.FC<InstallClusterProps> = ({ match }) => {
     null,
     CEPH_STORAGE_NAMESPACE,
   );
-
+  const [hasNoProvSC, setHasNoProvSC] = React.useState(false);
+  const [sc, isLoaded, err] = useK8sGet<ListKind<StorageClassResourceKind>>(StorageClassModel);
   const memoizedCSV = useDeepCompareMemoize(csv, true);
 
+  React.useEffect(() => {
+    if (!err && isLoaded) {
+      const isSCWithNoProv = sc?.items?.some(filterSCWithNoProv);
+      setHasNoProvSC(isSCWithNoProv);
+    }
+  }, [appName, err, isLoaded, ns, sc]);
+
```

Comment:
Don't see the need for `useEffect` nor the `hasNoProvSC` state.
Wouldn't this work? 
```
const hasNoProvSC = sc?.items?.some(filterSCWithNoProv);
```

Reply:
Yeah, I suppose this is extraneous now. Removing.